src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="style.css"
+      method="post"
+      data-qa="big"
+      class="bigForm"
+    >
+
+      <img
+        src="images/Search.svg"
+        alt="#"
+        class="iconBig"
+      >
+
+      <input
+        type="search"
+        placeholder="Try &quot;Los Angeles&quot"
+        class="bigOne"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      action="style.css"
+      method="post"
+      data-qa="small"
+      class="smallForm"
+    >
+
+      <img
+        src="images/Search.svg"
+        alt="#"
+        class="iconSmall"
+      >
+
+
+      <input
+        type="search"
+        placeholder="Try &quot;Los Angeles&quot"
+        class="smallOne"
```

Comment:
```suggestion
        class="smallInput"
```
use more descriptive names